app-crypt/tpm2-tools/tpm2-tools-5.2.ebuild

```
@@ -0,0 +1,71 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8,9} )
+inherit autotools bash-completion-r1 python-single-r1
+
+DESCRIPTION="Tools for the TPM 2.0 TSS"
+HOMEPAGE="https://github.com/tpm2-software/tpm2-tools"
+SRC_URI="https://github.com/tpm2-software/${PN}/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+fapi test"
+
+RDEPEND=">=app-crypt/tpm2-tss-3.0.1[fapi?]
+	dev-libs/openssl:=
+	net-misc/curl
+	sys-libs/efivar:=
+	${PYTHON_DEPS}"
+DEPEND="${RDEPEND}
+	test? (
+		app-crypt/swtpm
+		app-crypt/tpm2-abrmd
+		app-editors/vim-core
+		dev-tcltk/expect
+		dev-util/cmocka
+		dev-python/pyyaml
+	)"
+BDEPEND="virtual/pkgconfig
+	sys-devel/autoconf-archive"
+
+RESTRICT="!test? ( test )"
+REQUIRED_USE=" ${PYTHON_REQUIRED_USE} "
+
+# One of the tests fails without this patch. See
+# https://github.com/tpm2-software/tpm2-tools/issues/2767
+PATCHES=(
+	"${FILESDIR}/${PN}-5.1.1-no-efivar-automagic.patch"
+)
+
+src_prepare() {
+	sed -i \
+	"s/m4_esyscmd_s(\[git describe --tags --always --dirty\])/${PV}/" \
+	"${S}/configure.ac" || die
+	"${S}/scripts/utils/man_to_bashcompletion.sh"
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(use_enable fapi) \
+		$(use_enable test unit) \
+		--with-bashcompdir=$(get_bashcompdir) \
+		--enable-hardening
+}
+
+src_install() {
+	default
+
+	mv "${D}/$(get_bashcompdir)/tpm2_completion.bash" \
+	   "${D}/$(get_bashcompdir)/tpm2" || die
```

Even if unlikely to be installed on prefix, should still use `${ED}`

Could also optionally skip the double `get_bashcompdir` by using shell expansion:
```suggestion
	mv "${ED}"/$(get_bashcompdir)/tpm2{_completion.bash,} || die
```
This can feel a bit less clear in general but most people working on ebuilds are familiar with shell expansion and unbothered by it.

---

Done